### ISSUE TITLE
allow to change language without app restart

### ISSUE THREAD

**Issue by norbertschuler** [Plant-for-the-Planet-org/treecounter-app#2336]
**Describe the feature you'd like to see included in Plant-for-the-Planet App.**
Allow the app to change the language without restarting it, e.g. like this rather old package which doesn’t support auto-linking: 
https://www.npmjs.com/package/react-native-locale-listener

**Comment by Chiagozie-Udeh** [comment]
Still happening though and even when the App restarts, it doesn't change the language. It retains the language with which it opened @norbertschuler

**Comment by norbertschuler** [comment]
@Chiagozie-Udeh have you really killed the app or just put it in the background and opened it again?

**Comment by Chiagozie-Udeh** [comment]
@norbertschuler yes. I did both for various devices and it is not changing the language. Maybe you want to check again

**Comment by norbertschuler** [comment]
@Chiagozie-Udeh I tested once more with Browserstack. If you just put the app into the background and change the language, the Android cannot switch to the newly chosen language, but if you kill it and start it for new, it always used the new language chosen (on Samsung device the Browserstack tool to change the language does not work and you have to do it manually in the Settings app of the device). Kill the app with the Browserstack tool like this:

<img width="592" alt="Bildschirmfoto 2020-05-16 um 21 20 58" src="https://user-images.githubusercontent.com/1532418/82122013-272a8d00-97bb-11ea-9842-82d16c8711f6.png">

So I am afraid (1) I cannot reproduce your case and (2) this issue I created exactly for developing a solution to also change the language for the Android app without killing it. But I don't regard this as a priority issue as most users just want to use the app in the same language they use for the system on their devices.
